### PR TITLE
Add conf file path for rh-mongodb34 to mongod_conf

### PR DESCRIPTION
**Pull request by Rohan21Lobo**
/etc/opt/rh/rh-mongodb34/mongod.conf has same YAML format as of rh-mongodb26

### CODE REVIEW

File: insights/parsers/mongod_conf.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
 """
-MongodbConf - files ``/etc/mongod.conf``, ``/etc/mongod.conf`` and ``/etc/opt/rh/rh-mongodb26/mongod.conf``
-===========================================================================================================
+MongodbConf - files ``/etc/mongod.conf``, ``/etc/mongod.conf``, ``/etc/opt/rh/rh-mongodb26/mongod.conf`` and ``/etc/opt/rh/rh-mongodb34/mongod.conf``
+=====================================================================================================================================================
```
Comment (xiangce) [inline insights/parsers/mongod_conf.py:3]: Better make the title a brief introduction of the parser, but not to reflect each of the files, the following doc will do that in detail.

e.g.
```
MongodbConf - files - Configuration files for MongDB
```

File: insights/specs/default.py [hunk 2]
```diff
@@ -595,7 +595,8 @@ def md_device_list(broker):
     mongod_conf = glob_file([
                             "/etc/mongod.conf",
                             "/etc/mongodb.conf",
-                            "/etc/opt/rh/rh-mongodb26/mongod.conf"
+                            "/etc/opt/rh/rh-mongodb26/mongod.conf",
```
Comment (huali027) [inline insights/specs/default.py:598]: @Rohan21Lobo, from https://access.redhat.com/support/policy/updates/rhscl-rhel7,
 I found rh-mongodb26 is already retired, I think we can delete it. How do u think?

Comment (Rohan21Lobo) [reply insights/specs/default.py:598]: I think RHUI 3 still uses mongodb26. Don't have an instance to check as of now.

File: insights/parsers/mongod_conf.py (hunk 1 above)
Comment (Rohan21Lobo) [reply insights/parsers/mongod_conf.py]: Done, Please review.

Comment (xiangce) [reply insights/parsers/mongod_conf.py]: NIT: Sorry for the original typo, but it should be `MongoDB`

File: insights/specs/default.py (hunk 2 above)
Comment (xiangce) [reply insights/specs/default.py:598]: Yeap, it doesn't hurt to keep it here even `26` is retired.

File: insights/parsers/mongod_conf.py (hunk 1 above)
Comment (Rohan21Lobo) [reply insights/parsers/mongod_conf.py]: Fixed . ;) I copied blindly

**Pull request merged**